app/router.js

```diff
@@ -79,8 +79,10 @@ Router.map(function() {
         this.route('sessions-speakers');
         this.route('attendee');
       });
-      this.route('export');
-      this.route('settings');
+      this.route('settings', function() {
+        this.route('export');
+        this.route('options');
```

Comment:
I think rename file to danger-zone but keep route to options.
`Options - Show the heading "Danger Zone" here and add the current info of the settings page using the URL /settings/options`